test/unit/messaging/create-message.test.js

```diff
@@ -0,0 +1,29 @@
+const { BODY } = require('../../mocks/messaging/body')
+const { TYPE } = require('../../../app/constants/type')
+const { SOURCE } = require('../../../app/constants/source')
+
+const { createMessage } = require('../../../app/messaging/create-message')
+const { SESSION_ID } = require('../../mocks/messaging/session-id')
```

Comment:
Could we group the mocks, then the constants and then the { createMessage}? Would that be better?